src/Policy.ts

```diff
@@ -26,6 +27,13 @@ export type FailureReason<R> = { error: Error } | { value: R };
  * breakers, retries, etc.
  */
 export class Policy<ReturnType> {
+  /**
+   * Creates a bulkhead--a policy that limits the number of concurrent calls.
+   */
+  public static bulkhead(limit: number, queue: number = 0) {
```

Comment:
So the bulkhead provides both throttling and limiting of parallelism?

Reply:
Just limiting of parallelism, with an optional queue to avoid breaking immediately in bursty scenarios.